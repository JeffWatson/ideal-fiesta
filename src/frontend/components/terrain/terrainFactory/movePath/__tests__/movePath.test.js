import React from 'react';
import { shallow } from 'enzyme';
import MovePath from '../movePath';

describe('frontend/components/terrain/terrainFactory/movePath component', () => {
  const props = {
    direction: 'NORTH_END',
  };

  it('renders correctly', () => {
    const button = shallow(<MovePath {...props} />);

    expect(button).toMatchSnapshot();
  });
});
