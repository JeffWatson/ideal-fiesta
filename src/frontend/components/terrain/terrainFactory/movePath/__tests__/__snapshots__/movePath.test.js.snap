// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`frontend/components/terrain/terrainFactory/movePath component renders correctly 1`] = `
<div
  className="movePath NORTH_END"
/>
`;
